src/test/locale/tr.js

```diff
@@ -22,20 +22,20 @@ test('parse', function (assert) {
 
 test('format', function (assert) {
     var a = [
-            ['dddd, MMMM Do YYYY, h:mm:ss a',      'Pazar, Şubat 14\'üncü 2010, 3:25:50 pm'],
+            ['dddd, MMMM Do YYYY, h:mm:ss a',      'Pazar, Şubat 14 2010, 3:25:50 pm'],
             ['ddd, hA',                            'Paz, 3PM'],
-            ['M Mo MM MMMM MMM',                   '2 2\'nci 02 Şubat Şub'],
+            ['M Mo MM MMMM MMM',                   '2 2 02 Şubat Şub'],
             ['YYYY YY',                            '2010 10'],
-            ['D Do DD',                            '14 14\'üncü 14'],
-            ['d do dddd ddd dd',                   '0 0\'ıncı Pazar Paz Pz'],
-            ['DDD DDDo DDDD',                      '45 45\'inci 045'],
-            ['w wo ww',                            '7 7\'nci 07'],
+            ['D Do DD',                            '14 14 14'],
+            ['d do dddd ddd dd',                   '0 0 Pazar Paz Pz'],
+            ['DDD DDDo DDDD',                      '45 45 045'],
+            ['w wo ww',                            '7 7 07'],
             ['h hh',                               '3 03'],
             ['H HH',                               '15 15'],
             ['m mm',                               '25 25'],
             ['s ss',                               '50 50'],
             ['a A',                                'pm PM'],
-            ['[yılın] DDDo [günü]',                'yılın 45\'inci günü'],
+            ['[yılın] DDDo [günü]',                'yılın 45 günü'],
```

Comment:
This is incorrect. The old test is correct and we should keep that.

Reply:
is this the only spec that should be kept?

Reply:
As mentioned in the original issue earlier, one never uses "December 3rd", it is always "3 December" or rarely, "December 3". Ordinals are only used when you are saying something like "The 3rd week of the year", "The 5th day of the month" (this one actually uses a slightly different thing than ordinals but ordinals are also fine) or "the first Sunday of March".

Reply:
Did some further changes, can you re-check please? I am not sure if I covered everything.